packages/apps-config/src/endpoints/productionRelayPolkadot.ts

```diff
@@ -128,16 +128,6 @@ export function createPolkadot (t: TFunction): EndpointOption {
           Composable: 'wss://rpc.composable.finance'
         }
       },
-      {
-        info: 'crustParachain',
-        homepage: 'https://crust.network',
-        paraId: 2008,
-        isUnreachable: true,
-        text: t('rpc.polkadot.crust', 'Crust', { ns: 'apps-config' }),
```

Comment:
By removing this, it won't show up here -

![image](https://user-images.githubusercontent.com/1424473/155658771-8d602f87-2b63-418e-ac32-2c8d8ac6a6cf.png)

I'm not sure that is intended.